src/Mapper/BaseGroupedMapper.php

```diff
@@ -256,9 +256,9 @@ final public function hasOpenTab(): bool
      * @param string $tab            The tab the group belongs to, defaults to 'default'
      * @param bool   $deleteEmptyTab Whether or not the Tab should be deleted, when the deleted group leaves the tab empty after deletion
      *
-     * @return static
+     * @return $this
```

Comment:
WDYT @VincentLanglet ?

To me it didn't make a lot of sense to put here static and on the return type self. PHPStan and Psalm were happy BUT, static is a native return type on php 8.0 (this change was made when we were on 7.4 min), and when you try to change self for static it does not work, Psalm gets angry because static is not the same as $this.

Reply:
Also, we are forced to add either $this or static, if we remove this return, psalm complains. But as I said, to me makes more sense to add $this since static exists on php but does not have the same meaning.

Reply:
$this seems better than static indeed